kahuna/public/js/directives/gr-remember-scroll-top.js

```diff
@@ -0,0 +1,34 @@
+import angular from 'angular';
+import Rx from 'rx';
+import 'rx-dom';
+
+import {rxUtil} from '../util/rx';
+
+export const rememberScrollTop = angular.module('gr.rememberScrollTop', [
+    rxUtil.name
+]);
+
+rememberScrollTop.directive('grRememberScrollTop', ['subscribe$', function (subscribe$) {
+    const storageKey = 'gr:rememberScrollTop';
+
+    return {
+        restrict: 'A',
+        link: function(scope, element, attrs) {
+            // Only remember if attribute if truthy. This is an annoying workaround to the
+            // inability to add directives conditionally
+            if (scope.$eval(attrs.grRememberScrollTop)) {
```

Comment:
That won't work because the attribute has to always be there, but its value evaluated, due to how it's used in the panels, and the fact AngularJS doesn't support conditionally adding directives.